### PR TITLE
added section in our tyk pump on our tyk analytics fields

### PR DESCRIPTION
**Pull request by LLe27**
added details for each analytics data field for our schema sent from our tyk pump

### CODE REVIEW

**Review by tbuchaillot — APPROVED** [review]
LGTM

**Comment by letzya** [comment]
Sorry for suggesting it only now but it looks like the same use case as our other configs which we autogen from the code? @zalbiraw what do you think?

File: tyk-docs/content/tyk-stack/tyk-pump/tyk-analytics-schema.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+title: "Tyk Analytics Schema"
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-pump/tyk-analytics-schema.md]: Per discussion, change schemas to fields in the file name, title and everywhere else.

**Review by letzya — CHANGES_REQUESTED** [review]
Per discussion, change schemas to fields

File: tyk-docs/content/tyk-stack/tyk-pump/tyk-analytics-schema.md (hunk 1 above)
Comment (LLe27) [reply tyk-docs/content/tyk-stack/tyk-pump/tyk-analytics-schema.md]: Added the requested change @letzya. Please review thank you!

**Review by letzya — APPROVED** [review]

**Pull request merged**